apps/src/gamelab/GameLabSprite.js

```diff
@@ -541,6 +546,13 @@ var AABBops = function (p5Inst, type, target, callback, modifyPosition=true) {
           {
             other.position.sub(displacement);
 
+            // Here, expect the callee to treat the caller as immovable,
+            // and to simulate a totally inelastic collision,
+            // adopting the caller's velocity along the displacement axis
+            var thisVelocityAlongDisplacement = projectAontoB(this.velocity, displacement);
+            var otherVelocityAlongDisplacement = projectAontoB(other.velocity, displacement);
+            other.velocity.sub(otherVelocityAlongDisplacement).add(thisVelocityAlongDisplacement);
```

Comment:
I think in a near-future PR I'll be able to reimplement `a.displace(b)` in terms of `b.collide(a)`, which will probably break some existing apps in subtle ways but I think it's worth it.  That would deduplicate this logic with the above.